lib/mozart_fetcher/http_client.ex

```diff
@@ -1,11 +1,15 @@
 defmodule HTTPClient do
+  use ExMetrics
+
   def get(endpoint) do
-    cert = Application.get_env(:mozart_fetcher, :dev_cert_pem)
-    timeout = Application.get_env(:mozart_fetcher, :timeout)
+    ExMetrics.timeframe "function.timing.http_client.get" do
```

Comment:
is .timeframe recording how long get takes?

Reply:
@BradleyTesterSpratt take a look at the macro here:
https://github.com/bbc/ExMetrics/blob/master/lib/ex_metrics.ex#L19
See if you can figure it out if not ill try explain it how James explained it to me :)

Reply:
Have a little read of this: https://elixir-lang.org/getting-started/meta/macros.html

Reply:
yeah, i looked at that macro, thats where my question came from. :)